pipelines/ops/createuser/Jenkinsfile

```diff
@@ -0,0 +1,38 @@
+@Library('deploy-conf') _
+node() {
+    try {
+        String ANSI_GREEN = "\u001B[32m"
+        String ANSI_NORMAL = "\u001B[0m"
+        String ANSI_BOLD = "\u001B[1m"
+        String ANSI_RED = "\u001B[31m"
+        String ANSI_YELLOW = "\u001B[33m"
+
+        stage('checkout public repo') {
+            cleanWs()
+            checkout scm
+        }
+
+        ansiColor('xterm') {
+            stage('deploy'){
+                values = [:]
+                currentWs = sh(returnStdout: true, script: 'pwd').trim()
+                envDir = sh(returnStdout: true, script: "echo $JOB_NAME").split('/')[-3].trim()
+                module = sh(returnStdout: true, script: "echo $JOB_NAME").split('/')[-2].trim()
+                jobName = sh(returnStdout: true, script: "echo $JOB_NAME").split('/')[-1].trim()
+                ansiblePlaybook = "${currentWs}/ansible/createuser.yml"
+                ansibleExtraArgs = "--tags \"add-user\" --extra-vars \"remote=${params.instance_name} ssh_user=${params.username} password=${params.password} public_key=${params.public_key}\" --vault-password-file /var/lib/jenkins/secrets/vault-pass -v"
```

Comment:
@G33tha Create a Jenkins drop down parameter with choices as add-user and delete-user. Take the value for tags as ${params.option}